flow/rocev2_flow.yaml

```diff
@@ -0,0 +1,105 @@
+components:
+  schemas:
+    Rocev2.Flow:
+      description: |-
+        Rocev2 traffic flow configuration.
+      type: object
+      properties:
+        flow_infos:
+          $ref: '#/components/schemas/Rocev2.FlowInfo'
+          x-field-uid: 1
+        per_port_settings:
+          $ref: '#/components/schemas/Rocev2.PerPortSettings'
+          x-field-uid: 2
+    Rocev2.PerPortSettings:
+      description: |-
+        Rocev2 traffic flow configuration.
+      type: object
+      properties:
+        target_line_rate:
+          description: |-
+            Target Line Rate in percentage.
+          type: integer
+          format: uint64
+          x-field-uid: 1
+        inter_batch:
+          $ref: '#/components/schemas/Rocev2.InterBatch'
+          x-field-uid: 2
+    Rocev2.FlowInfo:
+      description: |-
+        Rocev2 Flow Info, per flow details.
+      type: object
+      properties:
+        name:
+          description: |-
+            Name of the flow group.
+          type: string
+          x-field-uid: 1
+        tx_endpoint: 
+          description: |-
+            Tx Endpoint. Empty for Rocev2.
+          type: string
+          x-field-uid: 2
+        rx_endpoint:
+          description: |-
+           Rx Endpoint. Empty for Rocev2.
+          type: string
+          x-field-uid: 3
+        rate:
+          description: |-
+           when line_rate mode is chosen, it is required that all flows with a tx endpoint on such a port
+            must choose line_rate mode. In this case, all the rdma flows will transmit data in interleaved manner based
+             on qp.message size as fast as possible but not exceeding the line_rate value configured.
+             If burst_mode is chosen, all flows with Tx endpoint on a port should be configured in burst_mode.
+              Then each rdma flows sends packets as per rocev2_flow_settings.message_size configuration
+               for fixed / infinite times with burst_gap time between each iteration.
+          type: string
+          x-enum:
+            line_rate:
+              x-field-uid: 1
+            burst_mode:
+              x-field-uid: 2
+          x-field-uid: 4
+        burst_mode:
+          description: >-
+            Traffic Burst Mode to applied in Rocev2 Traffic , Continuous or Fixed.
+          type: string
+          x-enum:
+            Continuous:
+              x-field-uid: 1
+            Fixed:
+              x-field-uid: 2
+          x-field-uid: 5
+        burst_count:
+          description: >-
+            Burst Count to applied in Rocev2 Traffic item.
+          type: integer
+          format: uint32
+          minimum: 0
+          maximum: 16777216
+          x-field-uid: 6
+    Rocev2.InterBatch:
+      description: |-
+        Rocev2 inter batch settigns applicable per port
+      type: object
+      properties:
+        inter_batch_period_unit:
+          description: |-
+            inter batch period units.
```

Comment:
inter -> Inter 
Needs more info on that all this means to packet generation by a tool implementing this,